### PR TITLE
NSFS | NC | CLI | add/update bucket policy of a bucket

### PR DESCRIPTION
**Pull request by romayalon**
### Explain the changes
1. Added --bucket policy flag to bucket add/update
2. fix bucket policy to be tested by account.name and not account.email when using nsfs nc
### Issues: Fixed #xxx / Gap #xxx
1. 

### Testing Instructions:
1. 

- [ ] Doc added/updated
- [ ] Tests added

### CODE REVIEW

File: src/cmd/manage_nsfs.js [hunk 1]
```diff
@@ -218,9 +230,18 @@ async function fetch_bucket_data(argv, config_root, from_file) {
             new_name: argv.new_name
         };
     }
+
+    if (argv.bucket_policy !== undefined) {
```
Comment (naveenpaul1) [inline src/cmd/manage_nsfs.js:234]: @romayalon are we passing policy in manage_nsfs arguments? I couldnt find it in help

Comment (romayalon) [reply src/cmd/manage_nsfs.js:234]: I forgot to add it to the help, will add it :) thanks!

**Review by naveenpaul1 — APPROVED** [review]
LGTM
small comment added

File: src/sdk/bucketspace_fs.js [hunk 2]
```diff
@@ -97,6 +106,7 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
         try {
             if (!access_key) throw new Error('no access key');
             const iam_path = this._get_access_keys_config_path(access_key);
+            console.log('ROMY: iam_path', iam_path);
```
[the remark this answers is not in the document]
Comment (romayalon) [reply src/sdk/bucketspace_fs.js]: thanks!

**Pull request merged**